ci/code_checks.sh

```diff
@@ -172,16 +172,6 @@ if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
         pandas.Period.asfreq \
         pandas.Period.now \
         pandas.arrays.PeriodArray \
-        pandas.Interval.closed \
```

Comment:
Looks like this shouldn't be included here?

Reply:
Yeah, something in the rebase went wrong. I’ve updated…